transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/ListTransferDestinationFragment.java

```diff
@@ -127,6 +127,12 @@ public void onClick(View v) {
         registerObservers();
     }
 
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        mListTransferDestinationViewModel.init();
```

Comment:
Sorry this is not enough, when we rotate the phone it will reinitialize again making unnecessary call to HW Platform please make necessary adjustment making sure we only initialize once
have a look on [Fragment Lifecycle](https://developer.android.com/guide/components/fragments) and [Activity Lifecycle](https://developer.android.com/guide/components/activities/activity-lifecycle) to compare with the existing behavior plus have a look on `ViewModelProviders.of(this, factory).get(ViewModel)` on how they made a cache or have a hint how to make view model survive on configuration changes

Reply:
Done